ui/src/main/webapp/app/table/table-component.js

```diff
@@ -272,6 +272,10 @@ function TableController(schema, results, table, events, common, types, time, cs
     var convertValue = function(name, value) {
         var parsedValue = value;
         if(parsedValue) {
+            var hllpCheck = 'com.clearspring.analytics.stream.cardinality.HyperLogLogPlus'
```

Comment:
I did have a different solution which I've just pushed up where I amend getShortValue() to return the value, via making a simple if check for whether the custom object had more than one field, if it only had one field it would not perform the .join operator which converts it to string.

`var customShortValue = function(fields, parts) {
        var showWithLabel = (fields.length !== 1)
        if (fields.length > 1) {
            return fields.map(function(field) {
                var layers = field.key.split('.');
                var customValue = parts;
                for (var i in layers) {
                    customValue = customValue[layers[i]];
                }
    
                customValue = service.getShortValue(customValue);
    
                if (showWithLabel) {
                    return field.label + ': ' + customValue;
                }
                return customValue;
            }).join(', ');
        }
        else {
            return fields.map(function(field) {
                var layers = field.key.split('.');
                var customValue = parts;
                for (var i in layers) {
                    customValue = customValue[layers[i]];
                }
    
                customValue = service.getShortValue(customValue);
    
                if (showWithLabel) {
                    return field.label + ': ' + customValue;
                }
                return customValue;
            })[0];
        }`